From: Jason Rhinelander <jason@imaginary.ca>
Date: Sat, 28 Mar 2020 12:22:07 -0300
Subject: Fully version library

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 247a127..601137b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -26,7 +26,7 @@ set(OXENMQ_VERSION_PATCH 3)
 set(OXENMQ_VERSION "${OXENMQ_VERSION_MAJOR}.${OXENMQ_VERSION_MINOR}.${OXENMQ_VERSION_PATCH}")
 message(STATUS "oxenmq v${OXENMQ_VERSION}")
 
-set(OXENMQ_LIBVERSION 0)
+set(OXENMQ_LIBVERSION ${OXENMQ_VERSION})
 
 
 if(CMAKE_PROJECT_NAME STREQUAL PROJECT_NAME)
